draft-ietf-rats-corim.md

```diff
@@ -1219,7 +1180,7 @@ The first `conditional-series-record` that successfully matches an ACS Entry ter
 If none of the series conditions match an ACS Entry, the triple is not matched,
 and no Endorsed values are accepted.
 
-The `authorized-by` value in `measurement-map` in the stateful environment, if present,
+The `"__authorities"` value in `measurement-map` in the stateful environment, if present,
```

Comment:
Can we define a type, then have the text refer to the type rather than the key which identifies the type?